test/docfx.Tests/DocsetTest.cs

```diff
@@ -166,4 +166,45 @@ public static async Task Build_With_File_Metadata_Files()
         Assert.Equal("fileMetadata1.json", a.GetProperty("meta1").GetString());
         Assert.Equal("fileMetadata2.json", b.GetProperty("meta1").GetString());
     }
+
+    [Fact]
+    public static async Task Build_With_RedirectUri_Files()
+    {
+        // Act
+        var outputs = await Build(new()
+        {
+            ["docfx.json"] =
+                """
+                {
+                    "build": {
+                        "content": [{ "files": [ "*.md" ] }],
+                        "dest": "_site",
+                        "exportRawModel": true
+                    }
+                }
+                """,
+            ["index.md"] =
+                """
+                ---
+                redirect_url: "redirected.html"
+                ---
+                # Dummy Heading1
+                """
+        });
+
+
+        // Assert
+        var result = outputs["index.html"]();
+        Assert.Equal(
+            """
+            <!DOCTYPE html>
+            <html>
+              <head>
+                <meta charset="utf-8">
+                <meta http-equiv="refresh" content="0;URL='redirected.html'">
+              </head>
+            </html>
+            """, result);
+
```

Comment:
Verify that:
- the generated sitemap does not contain redirections
- the content type in the generated manifest file is redirection

Reply:
Fix DocumentType constant definitions in this commit (https://github.com/dotnet/docfx/pull/8892/commits/17fa984d0c1c6c4731e36f0b5f2046469597c233)
And unit tests are added related to `redirect_url` and sitemap generation. (https://github.com/dotnet/docfx/pull/8892/commits/a09b8f83d7740f2e28768953ceb6f0a060750f04)